server/spec/requests/api/v1/api_users_spec.rb

```diff
@@ -81,6 +82,39 @@
         expect(json['message'])
           .to match(/param is missing or the value is empty: api_user/)
       end
+      end
+    end
+
+    context 'when the request is invalid as email already in use' do
+      let(:invalid_email) do
+        { 'email': 'Test_email@email.com', 'password_digest': 'password2', 'username': 'user12' } .to_json 
```

Comment:
This let block lacks an end.

Reply:
Let blocks have been changed in test files